src/css/style.nohero.css

```diff
@@ -42,10 +42,6 @@
     line-height: 2.9rem;
 }
 
-.prose code::after {
```

Comment:
Just spotted this has caused all inline code blocks to be rendered with a ` at the end - because this rule was previously forcing it to be removed.

<img width="286" alt="image" src="https://github.com/user-attachments/assets/56a0b7bc-0653-432b-8cf1-a9daae6988f3">

Reply:
Thanks for the heads up, Nick. I appreciate you pointing this out. Here's the PR that addresses the issue: https://github.com/FlowFuse/website/pull/2402